xdn-sapper-example/lib/cms.js

```diff
@@ -0,0 +1,74 @@
+import fetch from 'axios';
+
+const origin = 'https://moovweb-docs-xdn-examples-api-default.moovweb-edge.io'
+let apiUrl
+
+if (typeof window !== 'undefined') {
+  apiUrl = location.protocol + '//' + location.host + '/api'
```

Comment:
You should always add a build number to API urls to ensure that when the browser downloads a new client, it doesn't use API responses cached in the service worker by the previous version of the app.  Next.js getServerSideProps automatically does this, but other frameworks such as Sapper (and probably Nuxt, not sure) do not.

Reply:
I added a build timestamp defined in webpack config that is set in the env to reference. Getting the Sapper build ID at build- and run-time seemed a bit more complex as build.json isn't available to import during the build.